analysis/tests/src/expected/CompletionExpressions.res.txt

```diff
@@ -167,6 +167,14 @@ Completable: Cexpression CArgument Value[fnTakingRecord]($0)->recordField(nested
     "documentation": null,
     "insertText": "Some(${1:_})",
     "insertTextFormat": 2
+  }, {
+    "label": "Some({})",
+    "kind": 12,
+    "tags": [],
+    "detail": "otherRecord",
+    "documentation": null,
+    "insertText": "Some({$0})",
```

Comment:
is this intended in addition to the previous completion?

Reply:
Yes. Logic being that you might want to do one of three things:
- Add None
- Add Some but with any
- Add Some, plus whatever value for the nested type you're interested in

That's why I'm adding it in addition rather than fully replacing the completions.

Reply:
OK seems sensible.